env: "gazebo"
mode: 0 # INIT = 0 # TELEOP = 1 # TASK_CONTROL = 2 # JOINT_CONTROL = 3 # RL = 4 # MOVEIT = 5 # IDLE = 6
haptic_feedback: false
base_controller: "scaled_joint_trajectory_controller"
position_controller: "forward_position_controller"
velocity_controller: "forward_velocity_controller"
input_pos_gain: 0.00008
input_ori_gain: 0.00016
task_p_gain: 3
task_d_gain: 3
init_pose: [0.180, 0.531, 0.589, -1.544, 0.002, -0.120] # wrist_3_link 기준
init_joint_states: [-1.601372543965475, -1.3494799772845667, -2.0361130873309534, -1.3006231672108264, 1.5698880420405317, 0.09116100519895554]